src/modules/flow/http-server/http-server.c

```diff
@@ -1163,6 +1303,19 @@ blob_handle_response_cb(struct sol_flow_node *node,
     return 0;
 }
 
+/*
+   Only implemented to be used by send_sse_data()
+ */
+static int
+json_response_cb(struct http_data *mdata, struct sol_buffer *buf, bool json)
+{
+    if (!mdata->value.blob)
+        return 0;
+
+    return sol_buffer_append_bytes(buf,
```

Comment:
This function is used by the http-server/json and it's the json content itself. There's no need to convert to base64. However, this may be useful for the http-server/blob node.